Index: src/main/java/example/taskmanager/service/TaskService.java
===================================================================
diff --git a/src/main/java/example/taskmanager/service/TaskService.java b/src/main/java/example/taskmanager/service/TaskService.java
--- a/src/main/java/example/taskmanager/service/TaskService.java	
+++ b/src/main/java/example/taskmanager/service/TaskService.java	
@@ -5,7 +5,6 @@
 import example.taskmanager.dto.TaskDTO;
 import example.taskmanager.model.TaskEntity;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestBody;
